packages/components/src/internal/components/assay/RunDataPanel.tsx

```diff
@@ -38,12 +38,34 @@ import { Alert } from '../base/Alert';
 
 import { getContainerFilterForLookups } from '../../query/api';
 
+import { AssayDomainTypes } from '../../AssayDefinitionModel';
```

Comment:
nit: There is dead code in this file. `onTextChange` and `clearText` can be removed.